app/src/main/java/com/example/paulsuarez/a26_simple_android_apps_simple_counter/MainActivity.java

```diff
@@ -0,0 +1,208 @@
+package com.example.paulsuarez.a26_simple_android_apps_simple_counter;
```

Comment:
The spacing is a bit inconsistent and can make this class hard to follow. Try to get in the habit of sticking to a consistent convention for whitespace that balances compactness and ease of reading.